### PR TITLE
Add typings for AROptions & Echo flags

### PR DESCRIPTION
**Pull request by ds300**
Paired on this with @brainbicycle during knowledge share 🎓

### CODE REVIEW

File: src/lib/Components/Bidding/Screens/BidResult.tsx [hunk 1]
```diff
@@ -61,9 +61,8 @@ export class BidResult extends React.Component<BidResultProps> {
 
   exitBidFlow = async () => {
     if (this.props.bidderPositionResult.status === "LIVE_BIDDING_STARTED") {
-      const Emission = NativeModules.Emission || {}
```
Comment (ds300) [inline src/lib/Components/Bidding/Screens/BidResult.tsx:64]: this pattern was being used in a bunch of places. Might have been a remnant of the storybooks era, or an era before we had a jest mock for this.

File: src/lib/options.ts [hunk 2]
```diff
@@ -1,18 +0,0 @@
-import { NativeModules } from "react-native"
-const { Emission } = NativeModules
```
Comment (ds300) [inline src/lib/options.ts:2]: this file wasn't referenced anywhere

File: typings/emission.d.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { NativeModulesStatic } from "react-native"
+declare module "react-native" {
+  interface NativeModulesStatic {
+    Emission: {
+      userID: string
```
Comment (ds300) [inline typings/emission.d.ts:5]: this is called "module augmentation" in case you want to google the TS jargon :)

Comment (brainbicycle) [reply typings/emission.d.ts:5]: 📚 🧠 Thanks!

**Comment by ashfurrow** [comment]
Awesome stuff, thanks to you both!

**Pull request merged**